README.md

```diff
@@ -233,51 +218,9 @@ We recommend creating JavaScript-specific classes to bind to, prefixed with `.js
     }
     ```
 
-### Mixins
```

Comment:
We should keep this section but reword the description. In the wake of advising against `@extend`, this section should recommend using mixins to DRY up code if you are also serving gzip'd code. We should maintain a note that using mixins without arguments can lead to unnecessary code duplication if you aren't gzipping.